src/controllers/chatController.ts

```diff
@@ -1,26 +1,63 @@
-import { Elysia, t } from "elysia";
+import { Elysia } from "elysia";
 import { db } from '../dbConnect'
 import { chatNewMessagePOSTRequest } from "../utils/chatBodyPayloads";
 import { WebSocket } from "ws";
 import {  IChatTable } from "../models/database";
+import { appendFile } from "node:fs/promises";
 
 const chatController = new Elysia().group(
   "chat",
 
   (app) =>
-    app
+    app.onBeforeHandle(async ({request, body}) => {
+      const accessFile = Bun.file('chat-access.log')
+      if (await accessFile.exists() == false) {
+        appendFile('chat-error.log', `Access to ${request.url} from ${request.headers.get("host")} not logged properly. Cannot find caht-access.log file.`)
```

Comment:
Just check spelling: `caht-access`

Reply:
Thanks for catching that! Completely missed it 😅
Fixed